pxteditor/monaco-fields/field_tilemap.ts

```diff
@@ -8,15 +8,6 @@ namespace pxt.editor {
         protected tilemapName: string;
         protected isTilemapLiteral: boolean;
 
-        protected initAsync(): Promise<void> {
```

Comment:
was this just unused? can 'defaultTileset' below be removed as well?

Reply:
yep will fix